theme.json

```diff
@@ -188,127 +188,61 @@
       ],
       "fontSizes": [
         {
-          "name": "1",
-          "slug": "1",
-          "size": "12px"
+          "name": "XX Small",
+          "slug": "xx-small",
+          "size": "14px",
+          "fluid": false
         },
         {
-          "name": "2",
-          "slug": "2",
-          "size": "14px"
+          "name": "X Small",
+          "slug": "x-small",
+          "size": "16px",
+          "fluid": false
         },
         {
-          "name": "3",
-          "slug": "3",
-          "size": "16px"
-        },
-        {
-          "name": "4",
-          "slug": "4",
-          "size": "17px",
-					"fluid": {
-						"min": "17px",
-						"max": "18px"
-					}
-        },
-        {
-          "name": "5",
-          "slug": "5",
-          "size": "19px",
-					"fluid": {
-						"min": "19px",
-						"max": "21px"
-					}
-        },
-        {
-          "name": "6",
-          "slug": "6",
-          "size": "21px",
-					"fluid": {
-						"min": "21px",
-						"max": "23px"
-					}
+          "name": "Small",
+          "slug": "small",
+          "size": "18px",
+          "fluid": {
+            "min": "16px",
+            "max": "18px"
+          }
         },
         {
-          "name": "7",
-          "slug": "7",
-          "size": "23px",
-					"fluid": {
-						"min": "23px",
-						"max": "26px"
-					}
+          "name": "Medium",
+          "slug": "medium",
+          "size": "22x",
+          "fluid": {
+            "min": "18px",
+            "max": "22px"
+          }
         },
         {
-          "name": "8",
-          "slug": "8",
-          "size": "26px",
-					"fluid": {
-						"min": "26px",
-						"max": "30px"
-					}
+          "name": "Large",
+          "slug": "large",
+          "size": "24px",
+          "fluid": {
+            "min": "22px",
+            "max": "24px"
+          }
         },
         {
-          "name": "9",
-          "slug": "9",
+          "name": "X Large",
+          "slug": "xx-small",
```

Comment:
I believe this is the wrong slug here. This should be x-large in this new convention.

Reply:
This has been updated.